packages/plugins/whisper-parity/src/index.js

```diff
@@ -2,7 +2,7 @@ import { __ } from "embark-i18n";
 import { dappPath, canonicalHost, defaultHost } from "embark-utils";
 const constants = require("embark-core/constants");
 const API = require("./api.js");
-import { ws, rpc } from "./check.js";
+const PARITY_WHISPER_ERROR = "Parity's implementation of Whisper is not compatible with Whisper v6 (and therefore web3.js). Try changing the communication config to use '{client: \"geth\"}' instead.";
```

Comment:
Might make sense to have this only defined and exported in api.js so it can be reused here instead of defining the same error message in multiple places.